fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,36 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+from subprocess import call, TimeoutExpired
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+flags.DEFINE_enum('engine', 'libfuzzer', ['libfuzzer'],
```

Comment:
for simplicity since this isn't used i think let's remove it? in the end a fuzzing engine will be specified with an individual launcher script, correct?

Reply:
Make sense to me!